next.config.js

```diff
@@ -6,16 +6,6 @@ const nextConfig = {
   sassOptions: {
     includePaths: [path.join(__dirname, 'styles')],
   },
-  async redirects() {
-    return [
-      {
-        source: '/',
-        destination: '/home',
```

Comment:
When you remove this code, does it no longer direct to /home?

Reply:
@Tch4lla  after removing the code i rename home to index this way the base path shows the content in index.jsx which was previously name home.jsx